src/ux.h

```diff
@@ -5,6 +5,26 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+typedef struct {
+    // public key index for the address
+    uint32_t keyIndex;
+    // base-58 address
+    uint8_t b58_address[34];
+    // the starting index to be shown
+    uint8_t displayIndex;
+    // NULL-terminated string for display
+    uint8_t partialAddress[13];
```

Comment:
Is it the display size (12) plus one for null termination?

Reply:
Yep.

Reply:
Used the defined value + 1